### ISSUE TITLE
Is it possible to integrate with Flow Router?

### ISSUE THREAD

**Issue by ghost** [alxche/autoform-froala-editor#1]
Hello,

I would like to use your autoform-froala-editor, but I notice that it requires Iron Router.  I, however, am using FlowRouter.  Is it possible for me to get it to work with FlowRouter?  

Thanks,

Moshe

**Comment by alxche** [comment]
Yeah, this should be router-independent. Will push an update soon. Thank you for your participation!

**Comment by ghost** [comment]
Glad to hear that you can do an update.  Look forward to seeing it.

**Comment by JulianKingman** [comment]
I'm looking for flowrouter support, as well. Anything I can help with? Is a router even necessary? It looks like it could be accomplished with a plain old method, no?

**Comment by Amritpd** [comment]
hey there, just checking in if a package compatible with Flow Router has been released. Thanks
